### PR TITLE
Makefile

### PR DESCRIPTION
**Pull request by macosxgeek**
This file is needed for a reason ;-)

### CODE REVIEW

File: init.c [hunk 1]
```diff
@@ -0,0 +1 @@
+<test>
```
Comment (pawroman) [inline init.c:1]: This is not a makefile...

**Comment by pawroman** [comment]
I do agree that there's some makefile or setup file needed.

**Comment by pawroman** [comment]
Closing as this adds nothing of value

**Comment by macosxgeek** [comment]
For God's sake.. It was a joke pafson :]

**Pull request closed without merging**